glyphs-reader/src/font.rs

```diff
@@ -968,13 +971,21 @@ impl RawFont {
         Ok(())
     }
 
+    fn v2_to_v3_kerning(&mut self) -> Result<(), Error> {
+        if let Some(kerning) = self.other_stuff.remove("kerning") {
+            self.other_stuff.insert("kerningLTR".to_string(), kerning);
```

Comment:
amusingly in glyphsLib we ended up actually doing the opposite i.e. down-convert the kerningLTR/kerningRTL split dictionaries into a single kerning dict, like in was in glyphs v2 and in UFO.. the history of this is complicated as everything in font-land..
You may start from https://github.com/googlefonts/glyphsLib/pull/865

Reply:
I explicitly didn't handle the kerningRTL set yet. My intent wasthat this is valid for both v2 and v3 LTR kerning. I _think_ you are saying it is? Perhaps I should file a bug for kerningRTL.

Reply:
#333